src/Symfony/Component/Security/Http/Firewall/ExceptionListener.php

```diff
@@ -195,6 +195,10 @@ private function handleLogoutException(ExceptionEvent $event, LogoutException $e
     private function startAuthentication(Request $request, AuthenticationException $authException): Response
     {
         if (null === $this->authenticationEntryPoint) {
+            if (null !== $this->logger) {
+                $this->logger->notice(sprintf('No Authentication entry point configured, returning a %s HTTP response. Configure "entry_point" on the firewall if you want to modify the response.', Response::HTTP_UNAUTHORIZED));
```

Comment:
From a DX perspective, it would be nice to have a firewall name here. I think this is available through `$this->providerKey`? Could be added in either the message or the context.

Reply:
Good idea, added!